### PR TITLE
Change `_lsr` to `_behavioral_response`

### LINKED ISSUE #5258: Change `_lsr` to `_behavioral_response`
As we add other behavioral responses, like capital gains and deductions, we will want to name them consistently. We can start by changing, for example, `employment_income_before_lsr` to `employment_income_before_behavioral_response`.

### PR DESCRIPTION
**Pull request by Mshuning**
Fixes #5258

### CODE REVIEW

File: policyengine_us/variables/household/income/person/weekly_hours_worked.py [hunk 1]
```diff
@@ -30,14 +30,22 @@ class weekly_hours_worked_behavioural_response_income_elasticity(Variable):
...
 
     def formula(person, period, parameters):
-        original = person("weekly_hours_worked_before_lsr", period)
+        original = person(
+            "weekly_hours_worked_before_behavioral_response", period
+        )
         lsr = person("labor_supply_behavioral_response", period)
         if (lsr != 0).any():
-            income_effect = person("income_elasticity_lsr", period)
+            income_effect = person(
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/household/income/person/weekly_hours_worked.py:38]: could you add some unit tests for anything that is not covered here?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by PavelMakarchuk** [comment]
We will need to make some adjustments once #5287 merged

**Comment by Mshuning** [comment]
> We will need to make some adjustments once #5287 merged

is it already been merged?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,18 +5,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.137.3] - 2024-11-01 16:53:33
```
Comment (PavelMakarchuk) [inline CHANGELOG.md:8]: add back

File: CHANGELOG.md [hunk 3]
```diff
@@ -10132,8 +10120,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
-[1.137.3]: https://github.com/PolicyEngine/policyengine-us/compare/1.137.2...1.137.3
```
Comment (PavelMakarchuk) [inline CHANGELOG.md:10135]: add back

File: changelog.yaml [hunk 4]
```diff
@@ -8580,13 +8580,3 @@
     fixed:
     - remove uprating for SNAP variables that don't change
   date: 2024-10-31 18:57:07
-- bump: patch
```
Comment (PavelMakarchuk) [inline changelog.yaml:8583]: add back

File: policyengine_us/parameters/gov/states/co/tax/income/credits/sales_tax_refund/amount/amount.yaml [hunk 5]
```diff
@@ -1,13 +1,13 @@
 description: Colorado provides a flat sales tax refund credit of this amount.
 values:
-  2023-01-01: 800
+  2023-01-01: 661
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/co/tax/income/credits/sales_tax_refund/amount/amount.yaml:3]: change back

File: policyengine_us/tests/policy/baseline/gov/states/co/tax/income/credits/sales_tax_refund/co_sales_tax_refund.yaml [hunk 6]
```diff
@@ -51,20 +51,3 @@
     filing_status: SEPARATE
   output:
     co_sales_tax_refund: 234
-
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/co/tax/income/credits/sales_tax_refund/co_sales_tax_refund.yaml:54]: add back

File: policyengine_us/variables/gov/states/il/tax/income/exemptions/il_personal_exemption.py [hunk 7]
```diff
@@ -1,7 +1,4 @@
 from policyengine_us.model_api import *
-from policyengine_us.variables.gov.states.il.tax.income.exemptions.il_personal_exemption_eligibility_status import (
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/il/tax/income/exemptions/il_personal_exemption.py:2]: change back

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]
Revert the changes unrelated to the PR - it seems like there have been some merge confliacts

**Comment by PavelMakarchuk** [comment]
Duplicate of #5321

**Pull request closed without merging**